contents/tutorials/laudspeaker-posthog.md

```diff
@@ -0,0 +1,84 @@
+---
+title: How to automate customer journeys with PostHog with Laudspeaker
+sidebar: Docs
+showTitle: true
+author: ['Laudspeaker']
+date: 2023-30-01
+featuredImage: ../images/tutorials/banners/tutorial-1.png
+tags: ["apps", "site-apps"]
+---
+
+Laudspeaker is a cross-channel customer messaging platform and an open source alternative to platforms such as Customer.io, Braze, or Iterable. It enables you to create automated customer journeys so your users receive messages when they complete specific actions on your site. 
+
+Given that Laudspeaker and PostHog are both open source, and both track events across your product, integrating them can be especially powerful. You can use the [Laudspeaker Connector](/apps/laudspeaker-connector) for PostHog to trigger event customer journeys in Laudspeaker based on PostHog events, creating a wide variety of journeys, including:
+
+- Automated onboarding flows triggered by sign-up events
+- Ecommerce re-engagement journeys based on incomplete purchases
+- Renewal journeys when a users' subscription or trial is coming to an end
+
+In this tutorial we'll explain how to connect PostHog and Laudspeaker to create a simple customer journey. To get started you'll need a Laudspeaker account, and a PostHog instance.
+
+## Connecting PostHog and Laudspeaker
+
+### Step 1: Install the Laudspeaker connector
+
+To start with, go to the Apps section of your PostHog instance, search for and install the Laudspeaker Connector. 
+
+---image---
+
+To configure the connector correctly you will need to supply the Laudspeaker endpoint. If you are self-hosting Laudspeaker this is under your domain, otherwise it is `app.laudspeaker.com/events/posthog`
```

Comment:
```suggestion
To configure the connector correctly, you need to supply the Laudspeaker endpoint. If you are self-hosting Laudspeaker, this is under your domain, otherwise it is `app.laudspeaker.com/events/posthog`.
```